packages/cli-platform-android/src/commands/runAndroid/tryLaunchEmulator.ts

```diff
@@ -1,12 +1,12 @@
 import os from 'os';
 import execa from 'execa';
-import Adb from './adb';
+import adb from './adb';
 
 const emulatorCommand = process.env.ANDROID_HOME
   ? `${process.env.ANDROID_HOME}/emulator/emulator`
   : 'emulator';
 
-const getEmulators = () => {
+export const getEmulators = () => {
```

Comment:
I'd consider moving it to separate file to avoid unexpected circular deps and make it clearer what's shared (although one could say `export` says it all, and they would be right)